modules/ai/site-planner-connect/module.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Elementor\Modules\Ai\SitePlannerConnect;
+
+defined( 'ABSPATH' ) || exit;
+
+class Module {
+
+	public function __construct() {
+		add_action( 'rest_api_init', [ $this, 'on_rest_init' ] );
+		add_action( 'admin_menu', [ $this, 'register_menu_page' ], 100 );
+		add_filter('rest_prepare_application_password', function ( $response, $item, $request ) {
+			if ( $request->get_route() === '/wp/v2/users/me/application-passwords' && is_user_logged_in() ) {
+				$user = wp_get_current_user();
+				$response->data['user_login'] = $user->user_login;
+			}
+			return $response;
+		}, 10, 3);
+	}
+
+	public function on_rest_init(): void {
+		( new WpRestApi() )->register();
+	}
+
+	public function register_menu_page() {
+		add_submenu_page(
+			null, // Hidden page
+			'App Password Generator',
```

Comment:
just a question - this looks like a text to be rendered on the client page -  but this means that  it's not translated - I was under the impression that the editor

Reply:
Because it's a hidden page, the text is not visible to the user.